src/plugins/intel_gpu/src/plugin/sync_infer_request.cpp

```diff
@@ -934,7 +934,8 @@ std::vector<cldnn::event::ptr> SyncInferRequest::prepare_output(size_t output_id
     auto device_tensor_et = convert_to_supported_device_type(element_type);
     bool convert_needed = is_convert_required(device_tensor_et, element_type);
 
-    if (is_remote_tensor_impl && !convert_needed && !is_dynamic) {
+    // Even if the network is dynamic, if user tensor's shape is static, remote tensor can be set as plugin's output tensor
+    if (is_remote_tensor_impl && !convert_needed && !user_tensor_pshape.is_dynamic()) {
```

Comment:
Remote tensor can't be created with dynamic shape, so this `user_tensor_pshape.is_dynamic()` check is meaningless

Reply:
removed